src/main/kotlin/com/openlattice/graph/Graph.kt

```diff
@@ -78,9 +78,9 @@ class Graph(
     /* Create */
 
     override fun createEdges(keys: MutableSet<DataEdgeKey>): WriteEvent {
-        val partitionsInfoByEntitySet = partitionManager.getEntitySetsPartitionsInfo(keys.flatMap { listOf(it.src, it.dst, it.edge) }
+        val partitionsInfoByEntitySet = partitionManager.getEntitySetsPartitions(keys.flatMap { listOf(it.src, it.dst, it.edge) }
```

Comment:
Super trivial, but since `PartitionsInfo` is no longer a thing, maybe just call this `partitionsByEntitySet`?